201-vm-winrm-windows/azuredeploy.json

```diff
@@ -166,29 +176,29 @@
                 }
             },
             "resources": [
-              {
-                  "type": "Microsoft.Compute/virtualMachines/extensions",
-                  "name": "[concat(variables('vmName'),'/WinRMCustomScriptExtension')]",
-                  "apiVersion": "[variables('apiVersion')]",
-                  "location": "[resourceGroup().location]",
-                  "dependsOn": [
-                      "[concat('Microsoft.Compute/virtualMachines/', variables('vmName'))]"
-                  ],
-                  "properties": {
-       	           "publisher": "Microsoft.Compute",
-       	           "type": "CustomScriptExtension",
-                     "typeHandlerVersion": "1.4",
-                     "settings": {
-						 "fileUris": [ 
-									"https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-vm-winrm-windows/ConfigureWinRM.ps1",
-									"https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-vm-winrm-windows/makecert.exe",
-									"https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-vm-winrm-windows/winrmconf.cmd"
-								],
-						 "commandToExecute": "[concat('powershell -ExecutionPolicy Unrestricted -file ConfigureWinRM.ps1 ',variables('hostDNSNameScriptArgument'))]"
-                     }
-                 }
-              }
+                {
+                    "type": "Microsoft.Compute/virtualMachines/extensions",
+                    "name": "[concat(variables('vmName'),'/WinRMCustomScriptExtension')]",
+                    "apiVersion": "[variables('apiVersion')]",
+                    "location": "[resourceGroup().location]",
+                    "dependsOn": [
+                        "[concat('Microsoft.Compute/virtualMachines/', variables('vmName'))]"
+                    ],
+                    "properties": {
+                        "publisher": "Microsoft.Compute",
+                        "type": "CustomScriptExtension",
+                        "typeHandlerVersion": "1.4",
+                        "settings": {
+                            "fileUris": [
+                                "https://raw.githubusercontent.com/Azure/azure-quickstart-templates/master/201-vm-winrm-windows/ConfigureWinRM.ps1",
```

Comment:
please see the _artifactsLocation best practice: https://github.com/Azure/azure-quickstart-templates/blob/master/1-CONTRIBUTION-GUIDE/best-practices.md